src/customTypes/COTTypes/COTSurveyAPI.ts

```diff
@@ -1,10 +1,10 @@
-import { ObjectId } from "../custom"
-/* 
-Debiese haber un tipo general para el survey, api o no? Es una idea nada mas, no esta implementado.
-*/
-export declare type SurveyAPI = {
-    _id: ObjectId
-    code: ObjectId | string
-    display: string
-    subproperties: Object[] | ObjectId | string
-}
\ No newline at end of file
+import { ObjectId } from '@customTypes/custom'
+/* 
+Debiese haber un tipo general para el survey, api o no? Es una idea nada mas, no esta implementado.
+*/
+export declare type SurveyAPI = {
+    _id: ObjectId
+    code: ObjectId | string
+    display: string
+    subproperties: object[] | ObjectId | string
```

Comment:
```suggestion
    subproperties: Record<string, any>[] | ObjectId | string,
```
Usamos un tipado más fuerte en vez de object, a pesar de que usar any no es muy bueno.
https://itnext.io/use-typescript-record-types-for-better-code-ce1768c6cb53